src/index.html

```diff
@@ -22,6 +22,421 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div>
+      <header>
+        <div class="moyo_logo">
+          <a
+            href="index.html"
+            class="logo"
+          >
+            <img
+              src="images/logo.png"
+              alt="logo"
+            />
+          </a>
+        </div>
+        <div>
+          <nav class="nav">
+            <ul class="nav_list">
+              <li class="nav__item">
+                <a
+                  href="Apple"
+                  class="is-active nav_link"
+                >
+                  Apple
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="Samsung"
+                  class="nav_link"
+                >
+                  Samsung
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="Smartphones"
+                  class="nav_link"
+                >
+                  Smartphones
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="Laptops & Computers"
+                  class="nav_link"
+                  data-qa="nav-hover"
+                >
+                  Laptops & Computers
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="Gadgets"
+                  class="nav_link"
+                >
+                  Gadgets
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="Tablets"
+                  class="nav_link"
+                >
+                  Tablets
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="Photo"
+                  class="nav_link"
+                >
+                  Photo
+                </a>
+              </li>
+              <li class="nav__item">
```

Comment:
Add the blank line between the links 
```suggestion
              </li>
              
              <li class="nav__item">
```